### ISSUE TITLE
Unsupported class file major version 65

### ISSUE THREAD

**Issue by YannickRobin** [YannickRobin/easy-gradle-plugin#6]
When I execute `gradle test --stacktrace`, I have the following exception.

```
Caused by: BUG! exception in phase 'semantic analysis' in source unit '/Users/i303937/SAPDevelop/easy-extensions/helloworld/src/main/groovy/com/sap/cx/boosters/easy/extension/helloworld/easyrest/controller/HelloWorldController.groovy' Unsupported class file major version 65
```

`java -version` shows I'm using version 17.

`openjdk version "17.0.9" 2023-10-17
OpenJDK Runtime Environment Homebrew (build 17.0.9+0)
OpenJDK 64-Bit Server VM Homebrew (build 17.0.9+0, mixed mode, sharing)`

`gradle -version` shows the Gradle used JVM 21.0.1.

```
------------------------------------------------------------
Gradle 8.4
------------------------------------------------------------

Build time:   2023-10-04 20:52:13 UTC
Revision:     e9251e572c9bd1d01e503a0dfdf43aedaeecdc3f

Kotlin:       1.9.10
Groovy:       3.0.17
Ant:          Apache Ant(TM) version 1.10.13 compiled on January 4 2023
JVM:          21.0.1 (Homebrew 21.0.1)
OS:           Mac OS X 14.0 aarch64
```
`./gradlew -version` shows the Gradle used JVM 17.0.9.

**Comment by YannickRobin** [comment]
As a workaround, I added the section below to `build.gradle`
```
java {
    toolchain {
        languageVersion = JavaLanguageVersion.of(17)
    }
}
```

**Comment by apanwar** [comment]
fixed #65

**Comment by YannickRobin** [comment]
Another workaround is to run `gradle wrapper` and then `./gradlew test`
